jdi-light/src/main/java/com/epam/jdi/light/elements/complex/ListBase.java

```diff
@@ -34,24 +33,42 @@
 import static com.epam.jdi.tools.ReflectionUtils.getValueField;
 
 abstract class ListBase<T extends IListBase, A extends UISelectAssert>
-    extends UIBaseElement<A> implements IList<T>, ISetup, ISelector {
+        extends UIBaseElement<A> implements IList<T>, ISetup, ISelector {
 
     protected Class<?> initClass = UIElement.class;
     protected CacheValue<MapArray<String, T>> map = new CacheValue<>(MapArray::new);
     protected String titleFieldName = null;
 
+    public static JFunc1<Field[], String> GET_TITLE_FIELD_NAME = fields -> {
+        Field expectedField = LinqUtils.first(fields, f -> f.isAnnotationPresent(Title.class));
+        if (expectedField != null)
+            return expectedField.getName();
```

Comment:
please use {} for if

Reply:
done:
if (expectedField != null) {
            return expectedField.getName();
        }

Reply:
Will be done in the context of another request